backstory/src/main/java/com/google/sps/perspective/data/StoryDecision.java

```diff
@@ -41,4 +41,10 @@ public StoryDecision(String story) throws IllegalArgumentException {
   public String getStory() {
     return story;
   }
+
+  @Override
+  public boolean equals(Object other) {
```

Comment:
When overriding equals() it's also recommended to override hashCode():
https://www.geeksforgeeks.org/overriding-equals-method-in-java/

Reply:
I added a simple hashCode() method as well.

Reply:
Done